src/main/java/digital/vix/vertxdemo/dto/EndpointMetaDataDto.java

```diff
@@ -0,0 +1,34 @@
+package digital.vix.vertxdemo.dto;
+
+import digital.vix.vertxdemo.models.Endpoint;
+import digital.vix.vertxdemo.models.Information;
+
+public class EndpointMetaDataDto {
+
+	private Endpoint endpoint;
+	private Information metaData;
+
+	public EndpointMetaDataDto(Endpoint endpoint, Information metaData) {
+		super();
```

Comment:
No need to call a super here.

Reply:
That is just auto-generated. I believe it is there to help navigate to the super constructor easier.